### ISSUE TITLE
Feature: Filter by claimed on bundle and/or batch/mass claiming on bundle

### ISSUE THREAD

**Issue by JadianRadiator** [itchio/itch.io#1079]
Due to the sheer massive amount of the items in the Social Justice Bundle.

I am unable to really know whether or not I've really claimed all of the games.

And then there's new additions some-times seemingly getting inserted in the "middle" of the bundle,
and not at the end like I would hope for.

As such I would like to request either the ability to filter out items I've already claimed.

Or (more importantly) the ability to manually mass-claim all the items in a bundle I've purchased.

I fully agree with not auto-claiming everything in a bundle that large.

However said sheer bulk makes it difficulty to claim everything the bundle contains.

As well as making it near impossible to be sure whether you've claimed everything, or where to find new additions in it.

**Comment by thenoble1** [comment]
I am very frustrated sitting here trying to claim everything right now. The workaround in the linked issue doesn't even work anymore. itch.io give us a way to just claim all of these into our library. PLEASE.

**Comment by leafo** [comment]
@thenoble1 
> I am very frustrated sitting here trying to claim everything right now. The workaround in the linked issue doesn't even work anymore. itch.io give us a way to just claim all of these into our library. PLEASE.

It is not necessary to "claim" everything in the bundle. You already have access to everything immediately upon buying it. We recommend that you browse the bundle through on the payment receipt page, and when you intend to play something you click the download button to make it visible in your library. Hope that helps.

**Comment by JadianRadiator** [comment]
yes, you don't have to 'claim' anything to access through the web page.
However you *do* have to claim it to access it through the itch.io pc
client.

**Comment by JadianRadiator** [comment]
and I assume they would have announced it if they changed that.

**Comment by legowerewolf** [comment]
This should be bumped again now that the [Bundle for Ukraine](https://itch.io/b/1316/bundle-for-ukraine) is a thing.

**Comment by XargonDragon** [comment]
I would personally love the option to "flood my library" so that I can easily search all of my owned items. Going through every page of both the Ukraine bundle and the Racial Justice and Equality and individually clicking each one is torturous

**Comment by joshgiesbrecht** [comment]
Is this a thing that itch would be open to accepting pull requests on, if an implementation didn't flood the server with too many hits at once? It's still kind of irritating to find out that I've already owned an interesting game I was ready to wishlist for later, because when I look at the game's store page I can't tell that it's owned via a large bundle purchase.
